### ISSUE TITLE
[Bug]: mergo import path is wrong

### ISSUE THREAD

**Issue by flowchartsman** [testcontainers/testcontainers-go#1370]
### Testcontainers version

current

### Using the latest Testcontainers version?

Yes

### Host OS

any

### Host arch

any

### Go version

any

### Docker version

```shell
n/a
```

### Docker info

```shell
n/a
```

### What happened?

go get -u github.com/testcontainers/testcontainers-go
go: github.com/imdario/mergo@v1.0.0: parsing go.mod:
        module declares its path as: dario.cat/mergo
                but was required as: github.com/imdario/mergo
        restoring github.com/imdario/mergo@v0.3.16

### Additional information

In short, the author of mergo decided to [celebrate his 1.0 release](https://github.com/imdario/mergo/discussions/236) by moving the project to a vanity domain, breaking all direct and indirect includes, including this one.  So everyone needs to update now. PR incoming.

**Comment by mdelapenya** [comment]
Closing as per #1419
